app/models/publisher.rb

```diff
@@ -84,6 +84,10 @@ def verify_uphold
     save!
   end
 
+  def uphold_complete?
+    self.uphold_verified || !self.uphold_access_parameters.blank?
```

Comment:
minor: could the second condition also be `self.uphold_access_parameters.present?`

Reply:
Good catch. I'll fix that up.